.travis.yml

```diff
@@ -17,8 +17,11 @@ python:
 before_install:
     - git config --global user.email "OpenStack_TravisCI@f5.com"
     - git config --global user.name "Travis F5 Openstack"
-    - echo "$DOCKERHUB_TOKEN" | docker login -u "$DOCKERHUB_ID" --password-stdin
-    - docker pull f5devcentral/containthedocs:latest
+    - git clone https://github.com/F5Networks/docker-images.git
```

Comment:
Do not recommend to clone the whole repo, because it will fetch all branches, tags and files. If you add new image which are not required by this CI job, it will waste your network bandwidth.

Recommend to download your required image via raw file link, eg. https://github.com/F5Networks/docker-images/raw/master/ubuntu:xenial.tar.gz

Reply:
No problem, I will change it part. Thanks for your review.

Reply:
> Do not recommend to clone the whole repo, because it will fetch all branches, tags and files. If you add new image which are not required by this CI job, it will waste your network bandwidth.
> 
> Recommend to download your required image via raw file link, eg. https://github.com/F5Networks/docker-images/raw/master/ubuntu:xenial.tar.gz

No problem, I will change it part. Thanks for your review.